### PR TITLE
resolves #81

### LINKED ISSUE #81: Refactor Profile displays for Validation and Hardening pages
As more links are added these start to be messy and hard to follow.

 [ ] Make the rows into columns, with links in alphabetical order.
 [ ] Tighten margins.
 [ ] Put Cloud Service Provider category up front. Alphabetize the rest.
 [ ] Clarify 'Platforms' as 'Virtual Platforms.'
 [ ] Make Docker a Virtual Platform.

### PR DESCRIPTION
**Pull request by wdower**
Signed-off-by: Will Dower <wdower@mitre.org>

Make the columns prettier and resize better for mobile view.

### CODE REVIEW

**Comment by aaronlippold** [comment]
Small bug on the view port
Screen Shot 2020-06-05 at 11 23 01 AM

**Comment by aaronlippold** [comment]
Also, let's make sure the order of the groups are the same on the hardening and validation

**Comment by aaronlippold** [comment]
Also I am getting this in the console 

```
2chunk-vendors.c5f16458.js:7 TypeError: Cannot read property 'goTo' of undefined
    at fe.scrollBehavior (app.37d3298a.js:1)
    at Or.<anonymous> (chunk-vendors.c5f16458.js:954)
    at Array.<anonymous> (chunk-vendors.c5f16458.js:7)
    at ce (chunk-vendors.c5f16458.js:7)
ie @ chunk-vendors.c5f16458.js:7
re @ chunk-vendors.c5f16458.js:7
ee @ chunk-vendors.c5f16458.js:7
(anonymous) @ chunk-vendors.c5f16458.js:7
ce @ chunk-vendors.c5f16458.js:7
Promise.then (async)
oe @ chunk-vendors.c5f16458.js:7
de @ chunk-vendors.c5f16458.js:7
tr @ chunk-vendors.c5f16458.js:7
nr.update @ chunk-vendors.c5f16458.js:7
vt.notify @ chunk-vendors.c5f16458.js:7
set @ chunk-vendors.c5f16458.js:7
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
zt.updateRoute @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
r @ chunk-vendors.c5f16458.js:954
r @ chunk-vendors.c5f16458.js:954
Mt @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
r @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
Promise.then (async)
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
Vt @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
d @ chunk-vendors.c5f16458.js:954
r @ chunk-vendors.c5f16458.js:954
r @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ chunk-vendors.c5f16458.js:954
(anonymous) @ app.37d3298a.js:1
d @ chunk-vendors.c5f16458.js:954
r @ chunk-vendors.c5f16458.js:954
r @ chunk-vendors.c5f16458.js:954
Mt @ chunk-vendors.c5f16458.js:954
zt.confirmTransition @ chunk-vendors.c5f16458.js:954
zt.transitionTo @ chunk-vendors.c5f16458.js:954
e.push @ chunk-vendors.c5f16458.js:954
fe.push @ chunk-vendors.c5f16458.js:954
m @ chunk-vendors.c5f16458.js:954
ne @ chunk-vendors.c5f16458.js:7
n @ chunk-vendors.c5f16458.js:7
Qi.o._wrapper @ chunk-vendors.c5f16458.js:7
2chunk-vendors.c5f16458.js:7 TypeError: Cannot read property 'goTo' of undefined
    at fe.scrollBehavior (app.37d3298a.js:1)
    at Or.<anonymous> (chunk-vendors.c5f16458.js:954)
    at Array.<anonymous> (chunk-vendors.c5f16458.js:7)
    at ce (chunk-vendors.c5f16458.js:7)
```

**Comment by wdower** [comment]
Fixed the breakpoints issue, thanks.

There aren't any hardening profiles under "Application Logic" so that category doesn't display on the hardening page. Other than that the categories are in the same order on both validation and hardening pages (Eugene said CSPs should be up front and then alphabetical order after that).

Not seeing the goTo issue on local console, site preview for this PR, or live site.

**Comment by aaronlippold** [comment]
I wonder if we can make the background on the larger image transparent so
then it 'just works' in light and dark mode.
--------
Aaron Lippold

lippold@gmail.com

260-255-4779

twitter/aim/yahoo,etc.
'aaronlippold'
